lessons/Algebra2/exploring-linear-models/langs/en-us/pages/exploring-states.adoc

```diff
@@ -0,0 +1,57 @@
+= Exploring the States Dataset
+
+For this page, you'll need to open the @starter-file{alg2-states} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Click "Run" and type `states-table` into the Interactions Area on the right to see the dataset.
+
+@vspace{2ex}
+
+[.FillVerticalSpace, cols="^1a,^1a",options="header",stripes="none"]
+|===
+| What do you Notice about this dataset?
+| What do you Wonder about this dataset?
+|
+|
+|===
+
+@n Lines 13 and 14 *extract rows 0 and 1 from the table, and define them as `alabama` and `alaska`*. Type `alabama` into the Interactions Area. What do you get back? @fitb{}{@ifsoln{Just the Row for Alabama}}
+
+
+@n What columns do you think might be related to one another? (e.g. - is the number of veterans related the amount of land-area? Is the population in 2010 related to the population in 2020?) List three possible relationships below.
+
++a.+ I think that @fitb{25em}{} may be related to @fitb{}{}
+
++b.+ I think that @fitb{25em}{} may be related to @fitb{}{}
+
++c.+ I think that @fitb{25em}{} may be related to @fitb{}{}
+
+@n For each of the relationships you listed, make a scatter plot using those columns. What do you think each display tells you about the relationship between those two columns?
+
```

Comment:
suggested including the contract for scatterplots here.

Reply:
Fixed